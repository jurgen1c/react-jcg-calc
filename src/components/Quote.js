import React from 'react';

const Quote = () => (
  <div>
    <h1>
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding. â€“William Paul ThurstonHomeCalculatorQuote
    </h1>
  </div>
);

export default Quote;
