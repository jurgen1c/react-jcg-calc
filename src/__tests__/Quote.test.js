import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('Componenet renders correctly', () => {
  const component = renderer.create(
    <Quote />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
