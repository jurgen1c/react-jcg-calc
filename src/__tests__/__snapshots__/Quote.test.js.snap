// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componenet renders correctly 1`] = `
<div>
  <h1>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul ThurstonHomeCalculatorQuote
  </h1>
</div>
`;
